### DISCUSSION TITLE
Why doesn't it match the official opentelemetry schema?

### DISCUSSION THREAD

**Discussion by overtrue** [spatie/laravel-open-telemetry#36] [Q&A]
Thank you for creating this package, I'm using it and I realized that the structure of the Span class of this package doesn't match the official structure of open telemetry, is there any other consideration?

https://github.com/open-telemetry/opentelemetry-proto/blob/v1.1.0/opentelemetry/proto/trace/v1/trace.proto#L86-L283

Thanks.

**Comment by chick84** [comment]
超哥貌似这个能够给你答案 [https://github.com/spatie/laravel-open-telemetry/discussions/3](https://github.com/spatie/laravel-open-telemetry/discussions/3)
